policyengine_us/variables/gov/states/va/tax/income/deductions/itemized_deductions/va_prorate_fraction_head.py

```diff
@@ -0,0 +1,23 @@
+from policyengine_us.model_api import *
+
+
+class va_prorate_fraction_head(Variable):
```

Comment:
we can make this a general variable as it only relates to federal agi and not state

Reply:
move to folder household income person general